lib/content_scripts/website/pages/utils.css

```diff
@@ -6,6 +6,10 @@
   margin: auto 0 auto;
 }
 
+h4 + .anime-list-link-container {
+  margin-left: .5rem;
+}
+
```

Comment:
This is a UI improvement for the anime list display, didn't want to make a separate PR for this